src/catalog.py

```diff
@@ -270,14 +289,15 @@ def package(self, package, version):
 
 
 class FreeBSDCatalog(Catalog):
-    def __init__(self, old_path, new_path):
+    def __init__(self, old_path: None, new_path: str) -> None:
```

Comment:
`None` should be `Union[None, str]`, right?

Reply:
aka `Optional[str]`, yes.